clang/lib/Sema/SemaOverload.cpp

```diff
@@ -7757,7 +7762,9 @@ void Sema::AddMethodCandidate(
   Candidate.Viable = true;
 
   unsigned FirstConvIdx = PO == OverloadCandidateParamOrder::Reversed ? 1 : 0;
-  if (ObjectType.isNull())
+  if (IgnoreExplicitObject)
+    ;
```

Comment:
How bad is it to turn this into `if (!IgnoreExplicitObjec)` and wrap the other `else if` cases? I'm not keen on an empty if statement body.